.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug report related to mistral-finetune
+description: Submit a bug report that's related to mistral-finetune
+title: '[BUG: '
+labels: ['bug', 'triage']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: python-vv
+    attributes:
+      label: Python -VV
+      description: Run `python -VV` from your virtual environment
+      placeholder: Copy-paste the output (no need for backticks, will be formatted into code automatically)
+      render: shell
+    validations:
+      required: true
+  - type: textarea
+    id: pip-freeze
+    attributes:
+      label: Pip Freeze
+      description: Run `pip freeze` from your virtual environment
```

Comment:
I guess ideally we'd just get the versions of the pypi packages that are required here: https://github.com/mistralai/mistral-finetune/blob/main/requirements.txt 

Otherwise people will post a 100 liner snippet no?